_format/1.1/index.md

```diff
@@ -534,10 +534,31 @@ Within this object, a link **MUST** be represented as either:
 * <a id="document-links-link-object"></a>an object ("link object") which can
   contain the following members:
   * `href`: a string containing the link's URI.
+  * `params`: a [link parameter object](link-parameter-object).
   * `meta`: a meta object containing non-standard meta-information about the
     link.
   * Any link-specific target attributes described below.
 
+A <a id="document-links-link-parameter-object"></a>link parameter object **MUST** be
+represented as an object which can contain the following members:
+
+* `rel`: a string or an array of strings. Each value **MUST** be a valid link
+  relation type as defined in [RFC8288 Section 2.1](https://tools.ietf.org/html/rfc8288#section-3.3) to include both registered
+  and extension relation types. When the `rel` member is not present, it **SHOULD** be
+  interpreted to contain the parent link object's key name.
+* `anchor`: a string containing URI reference as specified in
+  [[RFC8288 Section 3.2]](https://tools.ietf.org/html/rfc8288#section-3.2). When the `anchor` member is not present and the link belongs
+  to a resource object or relationship object, it **MUST** be interpreted to contain
+  the URI of the resource that generated the current response document with the
+  addition of the [URI fragment](fragments) identifying the object of which the link is a
+  member.
+* Any other valid target attribute name as explained in [RFC8288 Section 2.2](https://tools.ietf.org/html/rfc8288#section-3).
```

Comment:
Maybe worth listing them in here for readability?

Reply:
I don't think we should. The spec just establishes a few valid target attributes, but any implementation can make their own.